sdk/daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/UpgradesIT.scala

```diff
@@ -120,7 +129,15 @@ class UpgradesIT extends AsyncWordSpec with AbstractScriptTest with Inside with
           _ <- run(
             clients,
             QualifiedName.assertFromString(s"${testCase.name}:main"),
-            inputValue = Some(Value.ValueText("canton")),
+            inputValue = Some(
+              Value.ValueRecord(
+                None,
+                ImmArray(
+                  (None, Value.ValueEnum(None, Name.assertFromString("Canton"))),
+                  (None, Value.ValueList(FrontStack(Value.ValueText(testCase.name)))),
+                ),
+              )
```

Comment:
Perhaps we should abstract this building into a function, so it easier to extend later